### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by Zlopez**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -15,3 +15,4 @@ fedmsg.d/hotness.py
 Vagrantfile
 *.retry
 .cache
+.vagrant/
```
Comment (bowlofeggs) [inline .gitignore]: Since this doesn't have to do with docs, I recommend putting it in it's own pull request.

File: README.rst [hunk 2]
```diff
@@ -1,75 +1,21 @@
...
-    $ python setup.py develop
+Development
+^^^^^^^^^^^
 
-And then run it with::
-
-    $ fedmsg-hub
-
-It should pick up the the-new-hotness consumer and start running.
+Contributions are welcomed, check out `contribution guide <https://the-new-hotness.readthedocs.io/en/stable/dev-guide.html#contribution-guidelines>_`.
```
Comment (bowlofeggs) [inline README.rst]: s/welcomed/welcome/
s/contribution guide/the contribution guidelines/

File: .gitignore (hunk 1 above)
Comment (Zlopez) [reply .gitignore]: Fixed

File: README.rst (hunk 2 above)
Comment (Zlopez) [reply README.rst]: Fixed

**Pull request merged**